virt/virtual_machines/vm_networking/virt-attaching-vm-to-external-network.adoc

```diff
@@ -0,0 +1,80 @@
+:_content-type: ASSEMBLY
+[id="virt-attaching-vm-to-external-network"]
+= Attaching a virtual machine to an external network
+include::_attributes/common-attributes.adoc[]
+:context: virt-attaching-vm-to-external-network
+
+toc::[]
+
+{VirtProductName} is installed with a default internal `pod` network. If you want to attach VMs to an external network, however, you must create a Linux _network bridge_.
+
+Depending on the type of networks you are using, it may be necessary to update your xref:../../../virt/node_network/virt-updating-node-network-config.adoc#virt-about-nmstate_virt-updating-node-network-config[node networking configuration].
+
+Once a network bridge is created, you can start virtual machines on the external network in the same manner as you start virtual machines attached to internal networks, by using PXE booting, for example.
+
+= About configuring network bridges
```

Comment:
Either "configuring" or "creating" will work but you should be consistent. Also, don't forget to add anchor IDs to headers.

Reply:
Only the first header can be level 1 (= ). All subsequent headers must be level 2 or lower (==, ===, etc.).

Reply:
We got rid of this heading in the comment above.....